projects/js-packages/charts/src/components/line-chart/stories/index.stories.tsx

```diff
@@ -39,6 +39,7 @@ Default.args = {
 	showLegend: false,
 	legendOrientation: 'horizontal',
 	withGradientFill: false,
+	margin: { top: 20, right: 40, bottom: 40, left: 20 },
```

Comment:
I notice if no margin is passed to the component, it goes back to breaking. So I guess the default values on the component aren't working as expected?

Reply:
Good question. I'm struggling to find a good way to fix this as well - basically we need a bigger margin for axis if they exist.

Reply:
For default axis orientations - left, it works fine because the left margin is 40.